### ISSUE TITLE
bug: teleportation or cancellation of walking animation when moving to the next map

### ISSUE THREAD

**Issue by theceltos** [AscensionGameDev/Intersect-Engine#2117]
### Description

There is an anomaly happening with the walking animation when changing maps, which only outside players can notice.

https://youtu.be/VoPmZEk5ABo

### Steps to Reproduce

1 - log in with two characters
2 - walk with one of the characters to the next map within another player's view.
3 - the other player will see the anomaly, but you will not.

### Version with bug

0.8.0.0

### Last version that worked well

Unknown

### Affected platforms

Windows, I was *not* able test on other platforms

### Duplicate Bug Check

- [X] This bug report is not a duplicate to the best of my knowledge.

**Comment by theceltos** [comment]
Has anyone else noticed this anomaly? :O

**Comment by Litorals** [comment]
Yes, I was seeing this problem, the animation restarts for other players when moving from one map to another when they are linked.

**Comment by Litorals** [comment]
Reviewing the problem, if the character uses an "idle" animation, when moving from a linked map to another by walking, the "idle" animation is displayed. This is only visible to other players on the map.

**Comment by Litorals** [comment]
Test: 

https://github.com/AscensionGameDev/Intersect-Engine/assets/132066040/5ae00a1b-3e01-4754-aa17-e6a3127954cf
